packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,105 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const ignoredOperations = [
+  /* missing route */
+  /* #5694 */
+  "getSyncCommitteeRewards",
+  "getBlockRewards",
+  "getAttestationsRewards",
+  "getDepositSnapshot", // Won't fix for now, see https://github.com/ChainSafe/lodestar/issues/5697
+  "getBlindedBlock", // #5699
+  "getNextWithdrawals", // #5696
+  "getDebugForkChoice", // #5700
+  /* #6080 */
+  "getLightClientBootstrap",
+  "getLightClientUpdatesByRange",
+  "getLightClientFinalityUpdate",
+  "getLightClientOptimisticUpdate",
+  "getPoolBLSToExecutionChanges",
+  "submitPoolBLSToExecutionChange",
+];
+
+const ignoredProperties: Record<string, IgnoredProperty> = {
+  /*
+   #5693
+   missing finalized
+   */
+  getStateRoot: {response: ["finalized"]},
+  getStateFork: {response: ["finalized"]},
+  getStateFinalityCheckpoints: {response: ["finalized"]},
+  getStateValidators: {response: ["finalized"]},
+  getStateValidator: {response: ["finalized"]},
+  getStateValidatorBalances: {response: ["finalized"]},
+  getEpochCommittees: {response: ["finalized"]},
+  getEpochSyncCommittees: {response: ["finalized"]},
+  getStateRandao: {response: ["finalized"]},
+  getBlockHeaders: {response: ["finalized"]},
+  getBlockHeader: {response: ["finalized"]},
+  getBlockV2: {response: ["finalized"]},
+  getBlockRoot: {response: ["finalized"]},
+  getBlockAttestations: {response: ["finalized"]},
+  getStateV2: {response: ["finalized"]},
+
+  /* 
+   #6168
+   /query/syncing_status - must be integer
+   */
+  getHealth: {request: ["query.syncing_status"]},
+
+  /**
+   * #6185
+   *  - must have required property 'query'
+   */
+  getBlobSidecars: {request: ["query"]},
+
+  /* 
+   #4638 
+   /query - must have required property 'skip_randao_verification'
+   */
+  produceBlockV2: {request: ["query.skip_randao_verification"]},
+  produceBlindedBlock: {request: ["query.skip_randao_verification"]},
+};
+
 const openApiJson = await fetchOpenApiSpec(openApiFile);
-runTestCheckAgainstSpec(openApiJson, routesData, reqSerializers, returnTypes, testDatas, {
-  // TODO: Investigate why schema validation fails otherwise
-  routesDropOneOf: ["produceBlockV2", "produceBlindedBlock", "publishBlindedBlock"],
-});
+runTestCheckAgainstSpec(
+  openApiJson,
+  routesData,
+  reqSerializers,
+  returnTypes,
+  testDatas,
+  {
+    // TODO: Investigate why schema validation fails otherwise (see #6187)
+    routesDropOneOf: [
+      "produceBlockV2",
+      "produceBlockV3",
+      "produceBlindedBlock",
+      "publishBlindedBlock",
+      "publishBlindedBlockV2",
+    ],
```

Comment:
Does it still fails even including `ignored properties`?

Reply:
This a different issue, unrelated to properties
- see https://github.com/ChainSafe/lodestar/issues/6187

Ignoring the whole route would solve it but this is better